File: Chapter13/MyBlog/BlazorWebAssembly/Client/JSInteropSamples/JSToStaticNET.razor.js
export async function setMessage() {
    const { getAssemblyExports } = await globalThis.getDotnetRuntime(0);
    var exports = await getAssemblyExports("BlazorWebAssembly.Client.dll");
    alert(exports.BlazorWebAssembly.Client.JSInteropSamples.JSToStaticNET.GetAMessageFromNET());
}

export async function showMessage() {
    await setMessage();
}

File: Chapter12/MyBlog/BlazorWebAssembly/Client/JSInteropSamples/JSToStaticNET.razor.cs
using System.Runtime.InteropServices.JavaScript;
using System.Runtime.Versioning;
namespace BlazorWebAssembly.Client.JSInteropSamples;

[SupportedOSPlatform("browser")]
public partial class JSToStaticNET
{
    [JSExport]
    internal static string GetAMessageFromNET()
    {
        return "This is a message from .NET";
    }

    [JSImport("showMessage", "jstonet")]
    internal static partial void ShowMessage();
}
